corehq/apps/app_manager/app_schemas/tests/test_schema.py

```diff
@@ -24,6 +22,9 @@
     patch('corehq.apps.app_manager.app_schemas.session_schema.is_usercase_in_use', MagicMock(return_value=False)),
     patch('corehq.apps.app_manager.app_schemas.case_properties.is_usercase_in_use', MagicMock(return_value=False)),
     patch('corehq.apps.app_manager.app_schemas.case_properties.get_per_type_defaults', MagicMock(return_value={})),
+    patch(
+        'corehq.apps.app_manager.app_schemas.case_properties.domain_has_privilege',
```

Comment:
I haven't checked this, but I think from the traceback in the failing test, you need to patch 'corehq.apps.app_manager.app_schemas.casedb_schema.domain_has_privilege'

Reply:
FWIW there's also a `@privilege_enabled` decorator that may be useful here

Reply:
Thanks for the suggestions. @esoergel I unfortuantely wasn't able to get the `@privilege_enabled` decorator to work. I think that it is because the tests are subclassed from `SimpleTestCase`, and so are unable to run `domain_has_privilege()` correctly.